packages/go_router/lib/src/builder.dart

```diff
@@ -77,7 +78,7 @@ class RouteBuilder {
             return _buildErrorNavigator(
                 context,
                 e,
-                Uri.parse(matchList.location.toString()),
+                Uri.parse(matchList.uri.toString()),
```

Comment:
can't this entire line be just `matchList.uri` ?

Reply:
nice catch